tests/lib/rules/no-deprecated-classes.js

```diff
@@ -15,6 +15,18 @@ tester.run('no-deprecated-classes', rule, {
     '<template><div class="multiple classes" /></template>',
     '<template><div class="justify-center" /></template>',
     '<template><v-layout text-center /></template>',
+    '<template><div class="text-h1" /></template>',
```

Comment:
Don't need a test for every possible value, all the other tests in here are just making sure the fixer works in every scenario (class, attribute, v-bind, regexp replacer etc.)